docs/ref/settings.txt

```diff
@@ -385,6 +385,11 @@ where ``reason`` is a short message (intended for developers or logging, not for
 end users) indicating the reason the request was rejected.  See
 :doc:`/ref/csrf`.
 
+The default implementation ``django.views.csrf.csrf_failure`` takes an additional
+keyword parameter ``template_name`` that defaults to ``'csrf_fail.html'``. If a
```

Comment:
Ouh, that is a typo of course. Thanks! I fixed that.

I went for the longer version, as it is consistent with the names of the setting and the view, I think introducing a short version could be a source of confusion.